NorthstarDLL/squirrel/squirrel.cpp

```diff
@@ -549,6 +549,25 @@ template <ScriptContext context> void SquirrelManager<context>::ProcessMessageBu
 	_call(m_pSQVM->sqvm, message.args.size());
 }
 
+ADD_SQFUNC(
+	"string",
+	NSGetModName,
```

Comment:
should call this NSGetCallingModName or something similar to indicate that it comes from stack rather than like, a parameter